crates/usvg-parser/src/converter.rs

```diff
@@ -602,6 +607,37 @@ pub(crate) fn convert_group(
     }
 }
 
+pub(crate) fn resolve_transform_origin(
```

Comment:
Let's rename it into `resolve_transform`. This method might include `transform-box` in the future after all.

Reply:
And let's make it part of `SvgNode` in converter.rs to have a less verbose code.

This way instead of:

```rust
resolve_transform_origin(node, state, node.attribute(AId::Transform).unwrap_or_default());
```

we would get just:

```rust
node.resolve_transform(AId::Transform, state);
```